src/current/cockroachcloud/network-authorization.md

```diff
@@ -91,3 +91,51 @@ Self-service VPC peering setup is not supported for {{ site.data.products.dedica
 If your cloud provider is AWS, you can use [AWS PrivateLink](https://aws.amazon.com/privatelink/) to securely connect your AWS application with your {{ site.data.products.dedicated }} cluster using a private endpoint. Like VPC Peering, a PrivateLink connection will prevent your traffic from being exposed to the public internet and reduce network latency. 
 
 Refer to: [Managing AWS PrivateLink for a Dedicated Cluster](aws-privatelink.html).
+
+## DB Console
+
+The DB Console provides details about your cluster and database configuration, and helps you optimize cluster performance.
+
+For the functionality offered by the DB Console, refer to: [DB Console Overview](../{{site.versions["stable"]}}/ui-overview.html)
+
+To access the DB Console, you must first authorize your current IP address:
+
+1. Visit your Dedicated cluster's IP allowlist page:
+
+    `https://cockroachlabs.io/cluster/{ cluster ID }/networking/allowlist`
+
+1. Click **Add Network**.
+
+1. Add your **Current Network**:
```

Comment:
This is ending with a colon, is there something missing underneath, or is it a typo?

Reply:
yup, the below steps are nested. i fixed the indentation, thanks for the catch!